src/webgpu/api/operation/command_buffer/copies.spec.ts

```diff
@@ -17,13 +18,217 @@ Test Plan:
   first copy from srcBuffer to a region of dstBuffer, then copy from another part of srcBuffer to
   another region of dstBuffer which have overlaps with the region of dstBuffer in the first copy
   and check the content of the whole dstBuffer to see the copies are done in correct order.
+2. CopyTextureToTexture
```

Comment:
We should probably split T2T and B2B to two different files in command_buffer/copies and put the b2t2t2b and similar tests in basic.spec.ts